app/assets/stylesheets/components/_buttons.scss

```diff
@@ -80,6 +80,49 @@
   }
 }
 
+.button--green {
+  background: $color-green-money;
+  border-color: $color-green-money;
+  color: $color-black;
+
+  &:active {
+    background-color: $color-green-money;
+    border-color: $color-green-money;
+  }
+
+  &:hover {
+    background-color: darken($color-green-money, 5%);
+    border-color: darken($color-green-money, 5%);
+  }
+}
+
+.button--intercom {
+  padding: $s10 !important;
+  display: flex;
+  max-width: 21rem;
+  align-items: center;
+  position: fixed;
+  top: initial !important;
+  right: 2rem;
+  bottom: 2rem;
+  z-index: 3;
+  margin: 0;
+  box-shadow: 0 4px 6px rgba(0, 0, 0, 0.30);
+
+  img {
+    padding-right: $s10;
+  }
+
+  @media screen and (max-width: $mobile-up) {
+    font-size: $font-size-15;
+    padding: $s5 !important;
+
+    img {
+      width: $s25;
+    }
+  }
```

Comment:
Made the button a little bit smaller on small device sizes to not take up a huge amount of space.

![Screen Shot 2021-07-26 at 12 25 52](https://user-images.githubusercontent.com/9101728/127035066-fe1fe7bb-2916-4654-9336-0c438870a303.png)

![Screen Shot 2021-07-26 at 12 25 45](https://user-images.githubusercontent.com/9101728/127035071-33f985f6-337f-4af0-b77d-49f199a4bf48.png)